ringpop.go

```diff
@@ -89,8 +89,7 @@ type Ringpop struct {
 		sync.RWMutex
 	}
 
-	logger log.Logger
-	log    log.Logger
+	moduleLogger *util.ModuleLogger
```

Comment:
Maybe `mLog`? It looks a bit like Hungarian notation, but I think calling methods outside of `log.Logger` on a variable called `log` may be a bit surprising.